### ISSUE TITLE
Fiche Mission/Programme en pdf

### ISSUE THREAD

**Issue by orovellotti** [PnX-SI/GeoNature-citizen#228]
Bonjour 

Je pense qu'il serait intéressant de pouvoir éditer une fiche mission imprimable avec un QR code vers la mission pour les offices du tourisme colonie de vacances et autre classe verte.

Merci 

![Sans titre](https://user-images.githubusercontent.com/1989428/89296714-027bde00-d663-11ea-9835-df512320f46c.png)

**Comment by camillemonchicourt** [comment]
Oui utile

**Comment by samuelpriou** [comment]
Cela serait également adaptés aux maisons de parc nationaux et à tous nos relais sur le terrain (refuges, OT, etc).

**Comment by lpofredc** [comment]
Oui, idéal pour la com'. Il y a cette lib JS qui permet de générer des QRCode https://davidshimjs.github.io/qrcodejs/
